app/scripts/modules/core/entityTag/entityUiTags.component.ts

```diff
@@ -182,6 +182,23 @@ class EntityUiTagsComponent implements ng.IComponentOptions {
   `;
 }
 
+export class EntityUiTagsWrapperComponent implements IComponentOptions {
```

Comment:
React only can map to `<` bindings, so the wrapper component takes `<` bindings and maps them down to `&` and `@`. It allows the callback functionality to work in both react and angular for the component and not have to change all the places we use the component in angular to not have to change from `"text"` to `"'text'"`.

@danielpeach used the pattern for `help-field` and I stole the idea from him.